doc/mob_skill_db.md

```diff
@@ -0,0 +1,223 @@
+ * [db/pre-re/mob_skill_db.conf](../db/pre-re/mob_skill_db.conf)
+ * [db/re/mob_skill_db.conf](../db/re/mob_skill_db.conf)
+
+--------------------------------------------------------------
+
+## Entry structure
+```
+	<Monster_Constant>: {
+		<Skill_Constant>: {
+			ClearSkills:
+			SkillLevel:
+			SkillState:
+			SkillTarget:
+			Rate:
+			CastTime:
+			Delay:
+			Cancelable:
+			CastCondition:
+			ConditionData:
+			val0:
+			val1:
+			val2:
+			val3:
+			val4:
+			Emotion:
+			ChatMsgID:
+		}
+	}
+```
+
+--------------------------------------------------------------
+
+## Field list
+
+Name             | Data type | Default value
+:--------------- | :-------- | :------------
+Monster_Constant | constant  | No default value
+Skill_Constant   | constant  | No default value
+ClearSkills      | boolean   | false
+SkillLevel       | int       | 1
+SkillState       | string    | "MSS_ANY"
+SkillTarget      | string    | "MST_TARGET"
+Rate             | int       | 1
+CastTime         | int       | 0
+Delay            | int       | 0
+Cancelable       | boolean   | false
+CastCondition    | string    | "MSC_ALWAYS"
+ConditionData    | int       | 0
+val0             | int       | 0
+val1             | int       | 0
+val2             | int       | 0
+val3             | int       | 0
+val4             | int       | 0
+Emotion          | int       | -1
+ChatMsgID        | int       | 0
+
+--------------------------------------------------------------
+
+## Field explanation
+
+### Monster_Constant
+The monster's name constant, found in [mob_db.conf](../db/re/mob_db.conf) as *SpriteName*.  
+There are 3 special constants for global skill assignment:  
+* **ALL_MOBS**: Add skills to all monsters.
+* **ALL_MOBS_BOSS**: Add skills to all boss monsters.
+* **ALL_MOBS_NONBOSS**: Add skills to all non-boss monsters.
+
+### Skill_Constant
+The skill's name constant, found in [skill_db.conf](../db/re/skill_db.conf) as *Name*.  
+Note: You can add multiple Skill_Constant blocks.
+
+### ClearSkills
+If set to `true`, all previously defined skills for this monster will be removed.
+
+### SkillLevel
+The skill level which should be used.  
+Minimum value is `1`. Maximum value is `mob_max_skilllvl` from [conf/map/battle/skill.conf](../conf/map/battle/skill.conf).
+
+### SkillState
+Defines in which state the monster is able to cast the skill.
+State         | Description
+:------------ | :----------
+MSS_ANY       | Monster is in any state except MSS_DEAD.
+MSS_IDLE      | Monster has no target and isn't walking.
+MSS_WALK      | Monster is walking.
+MSS_LOOT      | Monster is looting or walking to loot.
+MSS_DEAD      | Monster is dying.
+MSS_BERSERK   | Monster is attacking after starting the battle.
+MSS_ANGRY     | Monster is attacking after being attacked.
+MSS_RUSH      | Monster is following an enemy after being attacked.
+MSS_FOLLOW    | Monster is following an enemy without being attacked.
+MSS_ANYTARGET | Same as MSS_ANY but monster must have a target.
```

Comment:
```suggestion
### SkillState
Defines in which state the monster is able to cast the skill.
State         | Description
:------------ | :----------
MSS_ANY       | Monster is in any state except `MSS_DEAD`.
MSS_IDLE      | Monster has no target and isn't walking.
MSS_WALK      | Monster is walking.
MSS_LOOT      | Monster is looting or walking to loot.
MSS_DEAD      | Monster is dying.
MSS_BERSERK   | Monster is attacking after starting the battle.
MSS_ANGRY     | Monster is attacking after being attacked.
MSS_RUSH      | Monster is following an enemy after being attacked.
MSS_FOLLOW    | Monster is following an enemy without being attacked.
MSS_ANYTARGET | Same as `MSS_ANY` but monster must have a target.
```
maybe easier to read when using the codeblock? since they are codes/constants and values.